core/events/events.js

```diff
@@ -218,101 +303,99 @@ Blockly.Events.BumpEvent;
  * appear connected.
  * @const
  */
-Blockly.Events.BUMP_EVENTS = [
-  Blockly.Events.BLOCK_CREATE,
-  Blockly.Events.BLOCK_MOVE,
-  Blockly.Events.COMMENT_CREATE,
-  Blockly.Events.COMMENT_MOVE
-];
+const BUMP_EVENTS = [BLOCK_CREATE, BLOCK_MOVE, COMMENT_CREATE, COMMENT_MOVE];
+exports.BUMP_EVENTS = BUMP_EVENTS;
```

Comment:
Would it be evil to just say:
```
exports.BUMP_EVENTS = [exports.BLOCK_CREATE, exports.BLOCK_MOVE, exports.COMMENT_CREATE, exports.COMMENT_MOVE]
```
Then 27 different `const` statements could disappear.

Reply:
I don't think I quite understand the suggestion here, could you clarify?